src/OrbitGl/MultivariateTimeSeries.h

```diff
@@ -102,11 +98,11 @@ class MultivariateTimeSeries {
     min_ = std::min(min_, value);
   }
 
-  std::array<std::string, Dimension> series_names_;
+  const std::array<std::string, Dimension> series_names_;
   std::map<uint64_t, std::array<double, Dimension>> time_to_series_values_;
   double min_ = std::numeric_limits<double>::max();
   double max_ = std::numeric_limits<double>::lowest();
-  std::optional<uint8_t> value_decimal_digits_ = std::nullopt;
+  const uint8_t value_decimal_digits_;
```

Comment:
I like these `const`! Maybe we can also add `const` for the `value_unit_` 😃

Reply:
I will tackle this in a follow up PR